manifests/package.pp

```diff
@@ -22,17 +26,41 @@
     }
 
     'RedHat': {
+      $pkg_title = 'sensu'
+      $pkg_name = 'sensu'
+      $pkg_source = undef
+
       class { '::sensu::repo::yum': }
 
       $pkg_require = undef
     }
 
+    'windows': {
+      $repo_require = undef
+
+      $pkg_version = inline_template("<%= scope.lookupvar('sensu::version').sub(/(.*)\\./, '\\1-') %>")
+      $pkg_title = 'sensu'
+      $pkg_name = 'Sensu'
+      $pkg_source = "C:\\Windows\\Temp\\sensu-${pkg_version}.msi"
+      $pkg_require = "Archive[${pkg_source}]"
+
+      archive { $pkg_source:
```

Comment:
Sensu::plugin uses the wget module, does that work on windows?  If not, I'd probably want to refactor to only use one fetching method, but I'm fine with this one if it's documented as a windows dependency.